### ISSUE TITLE
adobe-acrobat-reader fails to install/upgrade to version 24.002.20964 

### ISSUE THREAD

**Issue by antivirtel** [Homebrew/homebrew-cask#181409]
### Verification

- [X] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).
- [X] I have retried my command with `--force`.
- [X] I ran `brew update-reset && brew update` and retried my command.
- [X] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [X] I have checked the instructions for [reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs).
- [X] I made doubly sure this is not a [checksum does not match / SHA256 mismatch](https://docs.brew.sh/Common-Issues#cask---checksum-does-not-match) error (do not open an issue before trying to open a PR to fix first).

### Description of issue

Hi,

After the latest upgrade of adobe-acrobat-reader (24.002.20964) it seems to have an issue upgrading/installing (username replaced to MY-USERNAME):

```bash
$ brew install adobe-acrobat-reader --force
==> Downloading https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2400220964/AcroRdrDC_2400220964_MUI.dmg
Already downloaded: /Users/MY-USERNAME/Library/Caches/Homebrew/downloads/5dfe7908af738d773086462c0d73a5fb0e18a698b75f4f7feee7c0db535f938a--AcroRdrDC_2400220964_MUI.dmg
==> Installing Cask adobe-acrobat-reader
==> Running installer for adobe-acrobat-reader with sudo; the password may be necessary.
installer: Package name is Adobe Acrobat Reader (Continuous)
installer: Installing at base path /
installer: The install failed. (The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance. An error occurred while running scripts from the package “AcroRdrDC_2400220964_MUI.pkg”.)
==> Purging files for version 24.002.20964 of Cask adobe-acrobat-reader
Error: Failure while executing; `/usr/bin/sudo -u root -E LOGNAME=MY-USERNAME USER=MY-USERNAME USERNAME=MY-USERNAME -- /usr/sbin/installer -pkg /usr/local/Caskroom/adobe-acrobat-reader/24.002.20964/AcroRdrDC_2400220964_MUI.pkg -target /` exited with 1. Here's the output:
installer: Package name is Adobe Acrobat Reader (Continuous)
installer: Installing at base path /
installer: The install failed. (The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance. An error occurred while running scripts from the package “AcroRdrDC_2400220964_MUI.pkg”.)
```

If I download the dmg directly: https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2400220964/AcroRdrDC_2400220964_MUI.dmg - it works fine (if I uninstall the old version first). Screenshot:
<img width="326" alt="Screenshot 2024-08-02 at 12 55 01" src="https://github.com/user-attachments/assets/0557301b-76fc-47b0-9e05-d95af850b97c">

Is brew uninstalling it first or is it auto handled? Please check.

Ps. last PR about this is: #180517 - the previous upgrade seemed to be fine: #177293

Thank you

### Command that failed

brew upgrade adobe-acrobat-reader or brew install adobe-acrobat-reader

### Output of command with `--verbose --debug`

```shell
# It was only ran after I manually upgraded the tool, but seem to have the same issue:
$ brew install adobe-acrobat-reader --verbose --debug
==> Downloading https://formulae.brew.sh/api/formula.jws.json
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --user-agent Homebrew/4.3.12-56-g635a776\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.5\)\ curl/8.6.0 --header Accept-Language:\ en --fail --remote-time --output /Users/MY-USERNAME/Library/Caches/Homebrew/api/formula.jws.json --location --time-cond /Users/MY-USERNAME/Library/Caches/Homebrew/api/formula.jws.json --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.12-56-g635a776\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.5\)\ curl/8.6.0 --header Accept-Language:\ en --fail --compressed --speed-limit 100 --speed-time 5 https://formulae.brew.sh/api/formula.jws.json
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 4287k  100 4287k    0     0  11.7M      0 --:--:-- --:--:-- --:--:-- 11.7M
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::NullLoader): loading adobe-acrobat-reader
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::NullLoader): loading adobe-acrobat-reader
==> Downloading https://formulae.brew.sh/api/cask.jws.json
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --user-agent Homebrew/4.3.12-56-g635a776\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.5\)\ curl/8.6.0 --header Accept-Language:\ en --fail --remote-time --output /Users/MY-USERNAME/Library/Caches/Homebrew/api/cask.jws.json --location --time-cond /Users/MY-USERNAME/Library/Caches/Homebrew/api/cask.jws.json --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.12-56-g635a776\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.5\)\ curl/8.6.0 --header Accept-Language:\ en --fail --compressed --speed-limit 100 --speed-time 5 https://formulae.brew.sh/api/cask.jws.json
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 1474k  100 1474k    0     0  5689k      0 --:--:-- --:--:-- --:--:-- 5693k
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading adobe-acrobat-reader
==> Cask::Installer#install
==> Printing caveats
==> Cask::Installer#fetch
==> Downloading https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2400220964/AcroRdrDC_2400220964_MUI.dmg
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.12-56-g635a776\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.5\)\ curl/8.6.0 --header Accept-Language:\ en --retry 3 --fail --location --silent --head https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2400220964/AcroRdrDC_2400220964_MUI.dmg
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.12-56-g635a776\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.5\)\ curl/8.6.0 --header Accept-Language:\ en --retry 3 --fail --location --silent --head --request GET https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2400220964/AcroRdrDC_2400220964_MUI.dmg
Already downloaded: /Users/MY-USERNAME/Library/Caches/Homebrew/downloads/5dfe7908af738d773086462c0d73a5fb0e18a698b75f4f7feee7c0db535f938a--AcroRdrDC_2400220964_MUI.dmg
==> Checking quarantine support
/usr/bin/env /usr/bin/xattr -h
/usr/bin/env /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -target x86_64-apple-macosx14 /usr/local/Homebrew/Library/Homebrew/cask/utils/quarantine.swift
==> Quarantine is available.
==> Verifying Gatekeeper status of /Users/MY-USERNAME/Library/Caches/Homebrew/downloads/5dfe7908af738d773086462c0d73a5fb0e18a698b75f4f7feee7c0db535f938a--AcroRdrDC_2400220964_MUI.dmg
/usr/bin/env /usr/bin/xattr -p com.apple.quarantine /Users/MY-USERNAME/Library/Caches/Homebrew/downloads/5dfe7908af738d773086462c0d73a5fb0e18a698b75f4f7feee7c0db535f938a--AcroRdrDC_2400220964_MUI.dmg
==> /Users/MY-USERNAME/Library/Caches/Homebrew/downloads/5dfe7908af738d773086462c0d73a5fb0e18a698b75f4f7feee7c0db535f938a--AcroRdrDC_2400220964_MUI.dmg is quarantined
==> Verifying checksum for '5dfe7908af738d773086462c0d73a5fb0e18a698b75f4f7feee7c0db535f938a--AcroRdrDC_2400220964_MUI.dmg'
/usr/bin/env hdiutil imageinfo -format /Users/MY-USERNAME/Library/Caches/Homebrew/downloads/5dfe7908af738d773086462c0d73a5fb0e18a698b75f4f7feee7c0db535f938a--AcroRdrDC_2400220964_MUI.dmg
==> Installing Cask adobe-acrobat-reader
==> Cask::Installer#stage
==> Extracting primary container
==> Using container class UnpackStrategy::Dmg for /Users/MY-USERNAME/Library/Caches/Homebrew/downloads/5dfe7908af738d773086462c0d73a5fb0e18a698b75f4f7feee7c0db535f938a--AcroRdrDC_2400220964_MUI.dmg
/usr/bin/env hdiutil attach -plist -nobrowse -readonly -mountrandom /private/tmp/homebrew-dmg20240802-46094-nxij5c /Users/MY-USERNAME/Library/Caches/Homebrew/downloads/5dfe7908af738d773086462c0d73a5fb0e18a698b75f4f7feee7c0db535f938a--AcroRdrDC_2400220964_MUI.dmg
/usr/bin/env find . -print0
/usr/bin/env mkbom -s -i /private/tmp/20240802-46094-ku3z79.list -- /private/tmp/20240802-46094-nfnw62.bom
/usr/bin/env ditto --bom /private/tmp/20240802-46094-nfnw62.bom -- /private/tmp/homebrew-dmg20240802-46094-nxij5c/dmg.bC5swd /private/tmp/homebrew-unpack20240802-46094-z8uf7z
/usr/bin/env diskutil info -plist /private/tmp/homebrew-dmg20240802-46094-nxij5c/dmg.bC5swd
/usr/bin/env diskutil eject /private/tmp/homebrew-dmg20240802-46094-nxij5c/dmg.bC5swd
/usr/bin/env hdiutil imageinfo -format /private/tmp/homebrew-unpack20240802-46094-z8uf7z/AcroRdrDC_2400220964_MUI.pkg
cp -p /private/tmp/homebrew-unpack20240802-46094-z8uf7z/AcroRdrDC_2400220964_MUI.pkg /usr/local/Caskroom/adobe-acrobat-reader/24.002.20964/AcroRdrDC_2400220964_MUI.pkg
==> Verifying Gatekeeper status of /Users/MY-USERNAME/Library/Caches/Homebrew/downloads/5dfe7908af738d773086462c0d73a5fb0e18a698b75f4f7feee7c0db535f938a--AcroRdrDC_2400220964_MUI.dmg
/usr/bin/env /usr/bin/xattr -p com.apple.quarantine /Users/MY-USERNAME/Library/Caches/Homebrew/downloads/5dfe7908af738d773086462c0d73a5fb0e18a698b75f4f7feee7c0db535f938a--AcroRdrDC_2400220964_MUI.dmg
==> /Users/MY-USERNAME/Library/Caches/Homebrew/downloads/5dfe7908af738d773086462c0d73a5fb0e18a698b75f4f7feee7c0db535f938a--AcroRdrDC_2400220964_MUI.dmg is quarantined
==> Propagating quarantine from /Users/MY-USERNAME/Library/Caches/Homebrew/downloads/5dfe7908af738d773086462c0d73a5fb0e18a698b75f4f7feee7c0db535f938a--AcroRdrDC_2400220964_MUI.dmg to /usr/local/Caskroom/adobe-acrobat-reader/24.002.20964
/usr/bin/env /usr/bin/xattr -p com.apple.quarantine /Users/MY-USERNAME/Library/Caches/Homebrew/downloads/5dfe7908af738d773086462c0d73a5fb0e18a698b75f4f7feee7c0db535f938a--AcroRdrDC_2400220964_MUI.dmg
/usr/bin/env /usr/bin/xargs -0 -- /bin/chmod -h u+w
/usr/bin/env /usr/bin/xargs -0 -- /usr/bin/xattr -w com.apple.quarantine 0381\;66acc2e9\;\;EBE43E51-3979-43DE-947C-A3DDE09A4B7D
==> Creating metadata directory: /usr/local/Caskroom/adobe-acrobat-reader/.metadata/24.002.20964/20240802115659.782
==> Creating metadata subdirectory: /usr/local/Caskroom/adobe-acrobat-reader/.metadata/24.002.20964/20240802115659.782/Casks
==> Installing artifacts
==> Installing artifact of class Cask::Artifact::Pkg
==> Running installer for adobe-acrobat-reader with sudo; the password may be necessary.
/usr/bin/sudo -u root -E LOGNAME=MY-USERNAME USER=MY-USERNAME USERNAME=MY-USERNAME -- /usr/sbin/installer -pkg /usr/local/Caskroom/adobe-acrobat-reader/24.002.20964/AcroRdrDC_2400220964_MUI.pkg -target / -verboseR
installer: Package name is Adobe Acrobat Reader (Continuous)
installer: Upgrading at base path /
installer:PHASE:Preparing for installation…
installer:PHASE:Preparing the disk…
installer:PHASE:Preparing Adobe Acrobat Reader (Continuous)…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Configuring the installation…
installer:STATUS:
installer:%2.200000
installer:PHASE:Validating packages…
installer:%4.436737
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:%9.887023
installer:PHASE:Writing files…
installer:%17.457448
installer:PHASE:Writing files…
installer:%23.766135
installer:PHASE:Writing files…
installer:%31.336559
installer:PHASE:Writing files…
installer:%40.168721
installer:PHASE:Writing files…
installer:%45.215670
installer:PHASE:Writing files…
installer:%51.524358
installer:PHASE:Writing files…
installer:%57.833045
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:%65.914403
installer:PHASE:Running package scripts…
installer:%66.544082
installer:PHASE:Running package scripts…
installer:%67.174875
installer:PHASE:Running package scripts…
installer:%67.805830
installer:PHASE:Running package scripts…
installer:%68.436774
installer:PHASE:Running package scripts…
installer:%69.068717
installer:PHASE:Running package scripts…
installer:%69.698393
installer:PHASE:Running package scripts…
installer:%70.331094
installer:PHASE:Running package scripts…
installer:%70.960303
installer:PHASE:Running package scripts…
installer:%71.591309
installer:PHASE:Running package scripts…
installer:%72.221950
installer:PHASE:Running package scripts…
installer:%72.343025
installer:PHASE:Running package scripts…
installer:PHASE:Running package scripts…
installer:PHASE:Running package scripts…
installer:PHASE:Running package scripts…
installer:PHASE:Validating packages…
installer:%97.750000
installer: The upgrade failed. (The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance. An error occurred while running scripts from the package “AcroRdrDC_2400220964_MUI.pkg”.)
==> Purging files for version 24.002.20964 of Cask adobe-acrobat-reader
Error: Failure while executing; `/usr/bin/sudo -u root -E LOGNAME=MY-USERNAME USER=MY-USERNAME USERNAME=MY-USERNAME -- /usr/sbin/installer -pkg /usr/local/Caskroom/adobe-acrobat-reader/24.002.20964/AcroRdrDC_2400220964_MUI.pkg -target / -verboseR` exited with 1. Here's the output:
installer: Package name is Adobe Acrobat Reader (Continuous)
installer: Upgrading at base path /
installer:PHASE:Preparing for installation…
installer:PHASE:Preparing the disk…
installer:PHASE:Preparing Adobe Acrobat Reader (Continuous)…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Waiting for other installations to complete…
installer:PHASE:Configuring the installation…
installer:STATUS:
installer:%2.200000
installer:PHASE:Validating packages…
installer:%4.436737
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:%9.887023
installer:PHASE:Writing files…
installer:%17.457448
installer:PHASE:Writing files…
installer:%23.766135
installer:PHASE:Writing files…
installer:%31.336559
installer:PHASE:Writing files…
installer:%40.168721
installer:PHASE:Writing files…
installer:%45.215670
installer:PHASE:Writing files…
installer:%51.524358
installer:PHASE:Writing files…
installer:%57.833045
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:PHASE:Writing files…
installer:%65.914403
installer:PHASE:Running package scripts…
installer:%66.544082
installer:PHASE:Running package scripts…
installer:%67.174875
installer:PHASE:Running package scripts…
installer:%67.805830
installer:PHASE:Running package scripts…
installer:%68.436774
installer:PHASE:Running package scripts…
installer:%69.068717
installer:PHASE:Running package scripts…
installer:%69.698393
installer:PHASE:Running package scripts…
installer:%70.331094
installer:PHASE:Running package scripts…
installer:%70.960303
installer:PHASE:Running package scripts…
installer:%71.591309
installer:PHASE:Running package scripts…
installer:%72.221950
installer:PHASE:Running package scripts…
installer:%72.343025
installer:PHASE:Running package scripts…
installer:PHASE:Running package scripts…
installer:PHASE:Running package scripts…
installer:PHASE:Running package scripts…
installer:PHASE:Validating packages…
installer:%97.750000
installer: The upgrade failed. (The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance. An error occurred while running scripts from the package “AcroRdrDC_2400220964_MUI.pkg”.)
/usr/local/Homebrew/Library/Homebrew/system_command.rb:396:in `assert_success!'
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11504/lib/types/private/methods/call_validation_2_7.rb:652:in `bind_call'
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11504/lib/types/private/methods/call_validation_2_7.rb:652:in `block in create_validator_procedure_fast0'
/usr/local/Homebrew/Library/Homebrew/system_command.rb:75:in `run!'
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11504/lib/types/private/methods/call_validation_2_7.rb:59:in `bind_call'
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11504/lib/types/private/methods/call_validation_2_7.rb:59:in `block in create_validator_method_fast0'
/usr/local/Homebrew/Library/Homebrew/system_command.rb:40:in `run'
/usr/local/Homebrew/Library/Homebrew/system_command.rb:44:in `run!'
/usr/local/Homebrew/Library/Homebrew/cask/artifact/pkg.rb:63:in `block in run_installer'
/usr/local/Homebrew/Library/Homebrew/cask/artifact/pkg.rb:76:in `with_choices_file'
/usr/local/Homebrew/Library/Homebrew/cask/artifact/pkg.rb:56:in `run_installer'
/usr/local/Homebrew/Library/Homebrew/cask/artifact/pkg.rb:32:in `install_phase'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:255:in `block in install_artifacts'
/usr/local/Homebrew/Library/Homebrew/cask/artifact_set.rb:10:in `each'
/usr/local/Homebrew/Library/Homebrew/cask/artifact_set.rb:10:in `each'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:248:in `install_artifacts'
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11504/lib/types/private/methods/call_validation.rb:270:in `bind_call'
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11504/lib/types/private/methods/call_validation.rb:270:in `validate_call'
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11504/lib/types/private/methods/_methods.rb:277:in `block in _on_method_added'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:115:in `install'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:246:in `block in run'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:235:in `each'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:235:in `run'
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11504/lib/types/private/methods/call_validation.rb:270:in `bind_call'
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11504/lib/types/private/methods/call_validation.rb:270:in `validate_call'
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11504/lib/types/private/methods/_methods.rb:277:in `block in _on_method_added'
/usr/local/Homebrew/Library/Homebrew/brew.rb:96:in `<main>'
```

### Output of `brew doctor` and `brew config`

```shell
$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed casks are deprecated or disabled.
You should find replacements for the following casks:
  adoptopenjdk


Warning: Some installed kegs have no formulae!
This means they were either deleted or installed manually.
You should find replacements for the following formulae:
  unrar

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  python-packaging
  terraform

Warning: Putting non-prefixed coreutils in your path can cause GMP builds to fail.


$ brew config
HOMEBREW_VERSION: 4.3.12-56-g635a776
ORIGIN: https://github.com/Homebrew/brew
HEAD: 635a776fce1b63b1be554ca39b0cdc5911b4bd95
Last commit: 2 hours ago
Core tap HEAD: b2d1bb8727afd801b00f5b916f4c23838dd9b7a2
Core tap last commit: 86 minutes ago
Core tap JSON: 02 Aug 11:56 UTC
Core cask tap HEAD: 006b82c4144da19153faf5528ad61f3b980356b0
Core cask tap last commit: 3 hours ago
Core cask tap JSON: 02 Aug 11:56 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: vim
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 12
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.3.4 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4/bin/ruby
CPU: dodeca-core 64-bit kabylake
Clang: 15.0.0 build 1500
Git: 2.46.0 => /usr/local/bin/git
Curl: 8.6.0 => /usr/bin/curl
macOS: 14.5-x86_64
CLT: 15.3.0.0.1.1708646388
Xcode: 15.4
```

### Output of `brew tap`

```shell
adoptopenjdk/openjdk
hashicorp/tap
homebrew/cask
homebrew/core
homebrew/services
```

**Comment by khipp** [comment]
Closing this since the cask has been bumped to version `24.002.21005` in https://github.com/Homebrew/homebrew-cask/pull/182266.

@antivirtel Feel free to open a new issue if the installation error persists.

**Comment by robinbowes** [comment]
I'm seeing exactly the same error with AcroRdrDC_2400320112_MUI.pkg
